README.md

```diff
@@ -14,7 +14,39 @@ Experimental checker for legal text that determines whether 18F can use a third-
 Requires Ruby 1.9+.
 
 1. Clone this repository and `cd` into the directory.
-1. Create a `urls.txt` file with a list of sites to check, one on each line. 18F, for example, is collecting a list of potential services in [this spreadsheet](https://docs.google.com/spreadsheets/d/180JGMG8O13_R9VxSDLYDWGg0JSWa3Higy911RS-PeNk/edit#gid=0) (private; use the `URL` column).
+1. Create a `urls.txt` file with a list of sites to check, one on each line. For example, 
+
+    ```bash
+	http://www.aha.io/product/integrations/github
+	https://apiary.io/
+	https://biterg.io/
+	http://circleci.com/
+	http://cloudcannon.com/
+	https://codeclimate.com/
+	https://codecov.io/
+	https://coveralls.io/
+	https://hub.docker.com/
+	https://floobits.com/
+	https://gemnasium.com/
+	https://desktop.github.com/
+	https://government.github.com/
+	https://gitter.im/
+	https://hakiri.io/
+	https://hakiri.io/
+	https://houndci.com/
+	https://huboard.com/
+	https://www.quantifiedcode.com/
+	https://readthedocs.org/
+	https://slack.com/
+	https://snap-ci.com/
+	https://snyk.io/
+	https://travis-ci.org/
+	https://travis-ci.com/
+	https://trello.com/
+	https://waffle.io/
+	http://wercker.com/
+	https://www.zenhub.io/
```

Comment:
Getting closer! 😉 The backticks and all these URLs being indented four spaces would be great.